nilearn/decoding/tests/test_decoder.py

```diff
@@ -230,10 +234,13 @@ def test_decoder_classification_string_label():
 def test_decoder_regression():
     X, y = make_regression(n_samples=200, n_features=125,
                            n_informative=5, noise=0.2, random_state=42)
+    X = StandardScaler().fit_transform(X)
+    y = (y - y.mean()) / y.std()
     X, mask = to_niimgs(X, [5, 5, 5])
-    for screening_percentile in [100, 20]:
-        model = DecoderRegressor(estimator='ridge', mask=mask,
-                                 screening_percentile=screening_percentile)
-        model.fit(X, y)
-        y_pred = model.predict(X)
-        assert_true(r2_score(y, y_pred) > 0.95)
+    for regressor in regressors:
```

Comment:
When the iterator & iterable look so similar, it is a good readability practice to make the iterator more conspicuous by ending it with an underscore. For example,   `for regressor_ in regressors:`